cargo +nightly-2020-11-19 install semverver fails

I get:
```
error[E0432]: unresolved import `rustc_ast::ast::Name`
 --> /Users/jrmuizel/.cargo/registry/src/github.com-1ecc6299db9ec823/semverver-0.1.46/src/mapping.rs:6:5
  |
6 | use rustc_ast::ast::Name;
  |     ^^^^^^^^^^^^^^^^^^^^ no `Name` in `ast`
```
and then a bunch of other errors.

--- a/README.md
+++ b/README.md
@@ -33,19 +33,12 @@ Then you can do:
 
 ```sh
 $ rustup component add rustc-dev llvm-tools-preview --toolchain nightly-2020-11-19
-$ cargo +nightly-2020-11-19 install semverver
+$ cargo +nightly-2020-11-19 install --git https://github.com/rust-lang/rust-semverver
 ```
 
 You'd also need `cmake` for some dependencies, and a few common libraries (if you hit
 build failures because of missing system-wide dependencies, please open an issue, so they
 can be added here).
-
-You can also install the newest version of the tool from git:
-
-```sh
-$ rustup component add rustc-dev llvm-tools-preview --toolchain nightly-2020-11-19
-$ cargo +nightly-2020-11-19 install --git https://github.com/rust-lang/rust-semverver
-```
 
 <details>
 

